examples/cognite_core/_api/cognite_cad_model.py

```diff
@@ -476,6 +488,7 @@ def list(
             | Sequence[str | tuple[str, str] | dm.NodeId | dm.DirectRelationReference]
             | None
         ) = None,
+        type_: Literal["CAD", "Image360", "PointCloud"] | list[Literal["CAD", "Image360", "PointCloud"]] | None = None,
```

Comment:
We need to make a type alias to avoid repeating the type. Too easy that someone updates one and forgets the other.